hw/ip/entropy_src/doc/programmers_guide.md

```diff
@@ -52,6 +52,13 @@ If firmware wants to use hardware conditioning for the inserted entropy bits, it
 1. Set the `FIPS_ENABLE` field in the [`CONF`](registers.md#conf) register to `kMultiBitBool4True`.
 1. Set `ES_TYPE` or `ES_ROUTE` (or both) in [`ENTROPY_CONTROL`](registers.md#entropy_control) to `kMultiBitBool4False`.
 
+Otherwise, if firmware does not want to use hardware conditioning for the inserted entropy bits, it has to do one of the two following points:
+1. Set the `FIPS_ENABLE` field in the [`CONF`](registers.md#conf) register to `kMultiBitBool4False`.
+1. Set `ES_TYPE` and `ES_ROUTE` in [`ENTROPY_CONTROL`](registers.md#entropy_control) to `kMultiBitBool4True`.
+
+For the first case, the entropy will be routed to software.
+For the second case on the other hand, the entropy will be routed to the hardware endpoints.
```

Comment:
I think it's the other way round. Also, it's maybe easier if you add the information directly to the list, i.e.:
```
Otherwise, if firmware does not want to use hardware conditioning for the inserted entropy bits, it has to do one of the two following points:
1. Set the `FIPS_ENABLE` field in the [`CONF`](registers.md#conf) register to `kMultiBitBool4False`.
   In this case, the unconditioned entropy is routed to the hardware endpoints.
1. Set `ES_TYPE` and `ES_ROUTE` in [`ENTROPY_CONTROL`](registers.md#entropy_control) to `kMultiBitBool4True`.
   In this case, the unconditioned entropy is not routed to the hardware endpoints but instead to the ["ENTROPY_READ_DATA"](...) register. 
```